website/blog/2024-07-24-cityblock-merge-jobs.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "How Cityblock Health adopted dbt Cloud's Merge Jobs to deploy new code faster"
```

Comment:
Should we call this like Practitioner Q&A: How Cityblock Health adopted dbt Cloud's Merge Jobs to deploy new code faster

Feels like we want something to note that this is an interview style dev blog